automation_tools/satellite6/upgrade/tasks.py

```diff
@@ -300,7 +300,12 @@ def create_rhevm_instance(instance_name, template_name, datacenter='Default',
                 vm_fqdn = rhevm_client.vms.get(
                     name=instance_name).get_guest_info().get_fqdn()
                 print('\t Instance FQDN : %s' % (vm_fqdn))
-                return vm_fqdn
```

Comment:
But may be with this change, it may not help with readability of code at the robotello-ci end.